### PR TITLE
Basic review while working on UpToCode

### LINKED ISSUE #42: Typo? - Reference to "User" in label
I think "User" in this label might be a typo. 

https://github.com/SuffolkLITLab/docassemble-ALAffidavitOfIndigency/blob/cda9766bfcf307fa739f48cd612242711ed92ea4/docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml#L420

### LINKED ISSUE #43: Punctuation fix on subquestion
"." should be "?" 
https://github.com/SuffolkLITLab/docassemble-ALAffidavitOfIndigency/blob/cda9766bfcf307fa739f48cd612242711ed92ea4/docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml#L532

### LINKED ISSUE #44: `id: regular job value` `show if` wrong?
Got "What does [user] get paid hourly?" when hourly is false, should be what currently appears if hourly is true "What does [user] get paid?"

Seems the show ifs are reversed

Also typo "What how much ..."

### PR DESCRIPTION
**Pull request by miabonardi**
fix #42 
fix #43 
fix #44

### CODE REVIEW

**Review by nonprofittechy — APPROVED** [review]
Looks good, will let Bryce add his review before merging as this is relevant to the launch of our new efiling

File: setup.py [hunk 1]
```diff
@@ -53,8 +54,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
       url='https://courtformsonline.org',
       packages=find_packages(),
       namespace_packages=['docassemble'],
-      install_requires=['docassemble.AssemblyLine', 'docassemble.MassAccess', 'docassemble.PovertyScale', 'docassemble.ALToolbox>=0.7.1', 'docassemble.ALMassachusetts>=0.1.2',
-          'docassemble.GithubFeedbackForm>=0.2.1'],
+      install_requires=[],
```
Comment (BryceStevenWilley) [inline setup.py:57]: I'll revert this change, we do need these packages to be installed for the Affidavit to work.

File: docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml [hunk 2]
```diff
@@ -417,7 +417,7 @@ id: education level
 question: |
   What is your highest level of education completed?
 fields:
-  - "User grade school completed": user_grade_school_completed
+  - "Grade school completed": user_grade_school_completed
```
Comment (BryceStevenWilley) [inline docassemble/ALAffidavitOfIndigency/data/questions/affidavit_of_indigency_supplement_body.yml:420]:
```suggestion
  - "Grade in school completed": user_grade_school_completed
```

Small suggestion

**Review by BryceStevenWilley — APPROVED** [review]
LGTM, thanks!

**Pull request merged**